packages/select/src/components/query-list/queryList.tsx

```diff
@@ -107,6 +108,13 @@ export interface IQueryListRendererProps<T> extends IProps {
      */
     handleKeyUp: React.KeyboardEventHandler<HTMLElement>;
```

Comment:
shouldn't this be `onKeyUp` for the prop? `handle[Event]` is for method names, right?

Reply:
this is a method passed to the renderer. hard to tell without context above.